doc/src/sgml/file-fdw.sgml

```diff
@@ -206,7 +206,7 @@
      as listing the column in <command>COPY</command>'s
      <literal>FORCE_NOT_NULL</literal> option.
 -->
-これはbooleanオプションです。
+これはブールオプションです。
```

Comment:
この上の184行で、

> `通常の値なしで指定される<command>COPY</command>オプションを有効にするには、そのようなオプションはすべてbooleanであるため、代わりにTRUEを渡すことができます。`

となっていて、こっちも変えないと不自然な気がします。

Reply:
> この上の184行で、
> 
> > `通常の値なしで指定される<command>COPY</command>オプションを有効にするには、そのようなオプションはすべてbooleanであるため、代わりにTRUEを渡すことができます。`
> 
> となっていて、こっちも変えないと不自然な気がします。

184行目は原文が
since all such options are Booleans.
です。合わせ方が難しいですね。

「それらは全てブールオプションであるため」と意訳するようなイメージになるでしょうか。